contracts/sophia.md

```diff
@@ -798,7 +822,7 @@ The block-chain environment available to a contract is defined in three name spa
 `Contract`, `Call`, and `Chain`:
 
 - `Contract.creator` is the address of the entity that signed the contract creation
-  transaction.
+  transaction. (Available from Fortuna release.)
```

Comment:
Contract creation transaction only? Or also generalized accounts attach transaction?

Reply:
The attach is considered a contract creation in this aspect.